tap-snapshots/test/precedence.tap.ts.test.cjs

```diff
@@ -62,6 +62,6 @@ Logical {
 }
 `
 
-exports[`test/precedence.ts > TAP > operator precedence > 9 + 8 * -7 > 6 / 5 and 4 == 3 or 2 - 1 > must match snapshot 2`] = `
+exports[`test/precedence.tap.ts > TAP > operator precedence > 9 + 8 * -7 > 6 / 5 and 4 == 3 or 2 - 1 > must match snapshot 2`] = `
```

Comment:
Don't forget to update test-generator accordingly! In fact, maybe just switch it to use vitest?